### PR TITLE
fix: storedSessions get returns object not string

### PR DESCRIPTION
**Pull request by cameronbraid**

### CODE REVIEW

File: packages-next/keystone/src/session/index.ts [hunk 1]
```diff
@@ -188,7 +188,8 @@ export function storedSessions({
     let isConnected = false;
     return {
       async get({ req, createContext }) {
-        let sessionId = await get({ req, createContext });
+        let data = await get({ req, createContext }) as any;
+        let sessionId = data?.sessionId
```
Comment (timleslie) [inline packages-next/keystone/src/session/index.ts:192]:
```suggestion
        const data = (await get({ req, createContext })) as { sessionId: string } | undefined;
        const sessionId = data?.sessionId;
```

File: packages-next/keystone/src/session/index.ts [hunk 2]
```diff
@@ -207,7 +208,8 @@ export function storedSessions({
         return start?.({ res, data: { sessionId }, createContext }) || '';
       },
       async end({ req, res, createContext }) {
-        let sessionId = await get({ req, createContext });
+        let data = await get({ req, createContext }) as any;
+        let sessionId = data?.sessionId
```
Comment (timleslie) [inline packages-next/keystone/src/session/index.ts:212]:
```suggestion
        const data = (await get({ req, createContext })) as { sessionId: string } | undefined;
        const sessionId = data?.sessionId;
```

File: packages-next/keystone/src/session/index.ts [hunk 3]
```diff
@@ -188,7 +188,9 @@ export function storedSessions({
     let isConnected = false;
     return {
       async get({ req, createContext }) {
-        let sessionId = await get({ req, createContext });
+        let data = await get({ req, createContext }) as any;
+        const data = (await get({ req, createContext })) as { sessionId: string } | undefined;
+        const sessionId = data?.sessionId;
```
Comment (timleslie) [inline packages-next/keystone/src/session/index.ts:193]:
```suggestion
        const data = (await get({ req, createContext })) as { sessionId: string } | undefined;
        const sessionId = data?.sessionId;
```

File: packages-next/keystone/src/session/index.ts [hunk 4]
```diff
@@ -207,7 +209,9 @@ export function storedSessions({
         return start?.({ res, data: { sessionId }, createContext }) || '';
       },
       async end({ req, res, createContext }) {
-        let sessionId = await get({ req, createContext });
+        let data = await get({ req, createContext }) as any;
+        const data = (await get({ req, createContext })) as { sessionId: string } | undefined;
+        const sessionId = data?.sessionId;
```
Comment (timleslie) [inline packages-next/keystone/src/session/index.ts:214]:
```suggestion
        const data = (await get({ req, createContext })) as { sessionId: string } | undefined;
        const sessionId = data?.sessionId;
```

**Review by timleslie — APPROVED** [review]
Good catch, thanks for the PR 🙏

**Comment by cameronbraid** [comment]
Thanks :)

**Pull request merged**